application/backend/src/main/resources/libs/bulma/components/title/title.html

```diff
@@ -20,4 +20,7 @@
   <h1 data-sly-element="${model.element}" class="${model.titleClasses @ join=' '}">
     ${model.text}
   </h1>
+  <h1 data-sly-element="${model.element}" class="${model.subtitleClasses @ join=' '}">
```

Comment:
I'm not sure if we don't need some data-sly-test to check if we should add this element at all. I mean, if we have a `addSubtitle` as false then we will add an empty `h1`?

Reply:
Fixed.